drivers/gpio/gpio_ite_it8xxx2.c

```diff
@@ -440,6 +575,7 @@ static const struct gpio_ite_cfg gpio_ite_cfg_##inst = {           \
 	.reg_gpcr = DT_INST_REG_ADDR_BY_IDX(inst, 1),              \
 	.reg_gpdmr = DT_INST_REG_ADDR_BY_IDX(inst, 2),             \
 	.reg_gpotr = DT_INST_REG_ADDR_BY_IDX(inst, 3),             \
+	.index = DT_INST_PROP(inst, index),                        \
```

Comment:
Change this to the following:
~~~C
        .index = (uint8_t)(DT_INST_REG_ADDR(inst) -                \
                           DT_REG_ADDR(DT_NODELABEL(gpioa))),      \
~~~

Reply:
Done.